### ISSUE TITLE
torchvision.transforms.Normalize has large absolute difference

### ISSUE THREAD

**Issue by jiannanWang** [pytorch/vision#6676]
### 🐛 Describe the bug

By definition, `torchvision.transforms.Normalize` should produce the same results if all input, std, and mean are divided or multiplied by the same number. When I test the API with the following input, I get the absolute difference up to 24978131.5 and the relative difference up to 3.5765e-08 for the float64 data type. Is this kind of difference expected? What's the threshold pytorch uses in tests to determine normal behavior versus buggy behavior?

Reproduce code:
```
import torch
import torchvision

input = torch.tensor([ 0.0000, 41.3108,  0.0000], dtype=torch.float64).view(3, 1, 1)
std = torch.tensor([61860.0, 3586.0, 60300.0])
mean = torch.tensor([4287419396147613455, -7376768754095287866, -6969696485275369284])

r1 = torchvision.transforms.Normalize(mean, std)(input)

input = input / 255.0
std = std / 255.0
mean = mean / 255.0

r2 = torchvision.transforms.Normalize(mean, std)(input)

print(r2 - r1)
print((r2-r1)/r1)
```
Output:
```
tensor([[[  588325.7422]],

        [[24978131.5000]],

        [[ 4133818.0781]]], dtype=torch.float64)
tensor([[[-8.4885e-09]],

        [[ 1.2142e-08]],

        [[ 3.5765e-08]]], dtype=torch.float64)
```

### Versions

```
Collecting environment information...
PyTorch version: 1.13.0.dev20220919+cpu
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: Ubuntu 18.04.6 LTS (x86_64)
GCC version: Could not collect
Clang version: Could not collect
CMake version: Could not collect
Libc version: glibc-2.17

Python version: 3.7.13 (default, Mar 29 2022, 02:18:16)  [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-4.15.0-176-generic-x86_64-with-debian-buster-sid
Is CUDA available: False
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] numpy==1.21.6
[pip3] torch==1.13.0.dev20220919+cpu
[pip3] torchaudio==0.13.0.dev20220919+cpu
[pip3] torchvision==0.14.0.dev20220919+cpu
[conda] numpy                     1.21.6                   pypi_0    pypi
[conda] torch                     1.13.0.dev20220919+cpu          pypi_0    pypi
[conda] torchaudio                0.13.0.dev20220919+cpu          pypi_0    pypi
[conda] torchvision               0.14.0.dev20220919+cpu          pypi_0    pypi
```

**Comment by pmeier** [comment]
Your values for `mean` are well past the point where `float64` can even differentiate between consecutive integers. My guess is that you are simply running into the limits what the `dtype` can represent, which in turn results in these subtle changes.

What is your use case for using these larges values for `mean`?

**Comment by jiannanWang** [comment]
Thanks for the response! We are doing fuzzing testing for extreme cases. Since this subtle difference is kind of expected, I would like to close this issue.
